content/applications/finance/fiscal_localizations/italy.rst

```diff
@@ -628,3 +628,40 @@ If the :abbr:`PA (Public Administration)` business doesn't reply within 15 days,
 the :abbr:`PA (Public Administration)` business directly, sending them both the invoice and the
 received deadline notification by email. You can make an arrangement with them and manually set the
 correct :guilabel:`SdI State` on your invoice.
+
+Point of Sale in Italy: Fiscal printer
+======================================
+
+Starting with Odoo 18.0, we introduced the support for Italian fiscal printers for your point of
+sale.
+These printers often come with the default setting to use HTTP. In order for such a printer to work
+with Odoo, that setting has to be changed to enable the printer to support HTTPS.
+
+EPSON FP90-III
+--------------
+
+To setup this printer, you will need the configuration software EpsonFPWizard or the keyboard
+connected to the printer. With the keyboard, you will then need to perform the following:
+
+#. Make sure you are on the default state of your fiscal printer
+#. Type **3333**
+#. Press **Chiave**
+#. Type **34**
+#. Press **Contante** until the screen displays **SSL**
+#. Press X to turn the value **0** into a **1**
+#. Press **Contante** 3 times to confirm
+#. Press **Chiave**
+
+After these steps are completed, you need to log in to the printer with the device that runs the
+:ref:`point of sale <sales/point_of_sale>` so it recognises your printer's certificate.
+
+#. Navigate to your printer by typing `https://<ip-of-your-printer>` in a web browser search bar. You
+will be presented with a security prompt.
+#. Click **advanced**
+#. Click **proceed**
```

Comment:
As we tend to be as precise as possible in the instructions, the intent for the following actions should be added to the top of the list.
Can you also add some details for steps 2 & 3 (the intent, the field or the window name)
```suggestion
#. To access the printer, open a web browser and enter `https://<ip-of-your-printer>` in the address
bar. A security message will then appear.
#. Click :guilabel:`advanced`
#. Click :guilabel:`proceed`
```

Reply:
@afma-odoo for this one i thought about inserting a picture of the webpage that appears (the one displaying the security message)
Would it be a valuable addition?
I took your comments into account for the rest

Reply:
@viso-odoo An image shouldn't add information, as text instructions should be self-explanatory without relying on images. Images can, of course, be included to help the user better understand the content, but they tend to become outdated quickly ;) So, they should only be used sparingly and only when they provide clear added value. If you think it should be useful, you can add it!